### PR TITLE
[YUNIKORN-951] Add perf-tool description into benchmarking tutorial page

### PR DESCRIPTION
**Pull request by 0yukali0**
Why we need this issue?
Describe how to use perf-tool in yunikorn release.

Issue link:
https://issues.apache.org/jira/browse/YUNIKORN-951

### CODE REVIEW

File: docs/performance/performance_tutorial.md [hunk 1]
```diff
@@ -355,6 +355,52 @@ scrape_configs:
...
+![Build-perf-tools](./../assets/perf-tutorial-build.png)
+
+### 3. Set test configuration
+Before start tests, check configuration whether meet your except.
+Default output path is `\tmp`, you can modify `common.outputrootpath` to change it.
+In each scenarios, it contains followings and we can set
+
+|	field			|			description					|
+| ----------------------------- | --------------------------------------------------------------------- |
+|	schedulerNames		|	List of scheduler will run these cases 				|
```
Comment (yangwwei) [inline docs/performance/performance_tutorial.md:388]: you can actually set 2 scheduler names? I wasn't aware of that
I think you need to add some more description to explain this a bit more

File: package.json [hunk 2]
```diff
@@ -12,7 +12,7 @@
   "dependencies": {
     "@docusaurus/core": "^2.0.0-beta.15",
     "@docusaurus/preset-classic": "^2.0.0-beta.15",
-    "@docusaurus/theme-search-algolia": "^2.0.0-beta.15",
+    "@docusaurus/theme-search-algolia": "^2.0.0-beta.17",
```
Comment (yangwwei) [inline package.json:15]: this change seems unrelated to this PR, can we skip this?

File: docs/performance/performance_tutorial.md [hunk 3]
```diff
@@ -355,6 +355,52 @@ scrape_configs:
...
+|	showNumOfLastTasks	|	Show the last tasks in scheduling				|
```
Comment (yangwwei) [inline docs/performance/performance_tutorial.md:389]: I feel this description isn't accurate. can you elaborate more?

**Review by yangwwei — CHANGES_REQUESTED** [review]

File: docs/performance/performance_tutorial.md [hunk 4]
```diff
@@ -355,6 +355,52 @@ scrape_configs:
 
 Once the environment is setup, you are good to run workloads and collect results. YuniKorn community has some useful tools to run workloads and collect metrics, more details will be published here.
 
+### 1. Scenarios 
+In performance tools, there are three types of tests and feedbacks.
+
+|	test type	|						description						|	diagram	|  		log		|
+| ---------------------	| -----------------------------------------------------------------------------------------------------	| ------------- | ----------------------------- |
+|	e2e test	|	Simulate and record the time in each steps							|	none	|	exist(QPS, timecost)	|
```
[the remark this answers is not in the document]
Comment (0yukali0) [reply docs/performance/performance_tutorial.md:363]: Ok, i will do it.

File: docs/performance/performance_tutorial.md (hunk 1 above)
Comment (0yukali0) [reply docs/performance/performance_tutorial.md:388]: Yes.
In throughput and fairness cases, this tool can run certain scheduler we require,
It will change scheduler name in pod spec field.
I will explain fields in each case separately in next commit.

File: package.json (hunk 2 above)
Comment (0yukali0) [reply package.json:15]: Ok

File: docs/performance/performance_tutorial.md (hunk 3 above)
Comment (0yukali0) [reply docs/performance/performance_tutorial.md:389]: Ok.

File: docs/performance/performance_tutorial.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (yangwwei) [reply docs/performance/performance_tutorial.md]: thanks, any updates?

[the remark this answers is not in the document]
Comment (0yukali0) [reply docs/performance/performance_tutorial.md]: Hi @yangwwei , Queue fairness isn't supported in this perf-tool now.
Maybe we can create a issue to trace it?
I think we can implement it based on node-fairness in perf-tools.
1.node capacity -> queue resource
2.change (allocated resource of pods*10/node allocatable resource) to (resource of pods according to queue/queue )

For now, i am going to add some screenshots and description to show how to set step by step.

[the remark this answers is not in the document]
Comment (yangwwei) [reply docs/performance/performance_tutorial.md]: Oh, I see, maybe that wasn't included in this package. I remember we have some results here in the past: https://yunikorn.apache.org/docs/0.8.0/performance/evaluate_perf_function_with_kubemark#resource-fairness-between-queues. It's fine, we don't need to address this now. Thanks for checking this.

**Comment by yangwwei** [comment]
hi @0yukali0 have you done the updates? I can take a look again if all my previous comments were addressed. Thanks!

**Comment by 0yukali0** [comment]
Hi @yangwwei , i update new one.
I also update some screenshots and configuration parts in this commit.

**Review by yangwwei — APPROVED** [review]
Thanks, looks good. Merging this to the master.
